### PR TITLE
Add tests for VM NIC binding options

### PR DESCRIPTION
**Pull request by rhrazdil**
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-2073
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-2074

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/utils/utils.ts [hunk 1]
```diff
@@ -2,12 +2,19 @@
 import { execSync } from 'child_process';
 import * as _ from 'lodash';
 import { $, $$, ElementFinder, browser, by, ExpectedConditions as until } from 'protractor';
-import { appHost } from '../../../../../integration-tests/protractor.conf';
+import { appHost, testName } from '../../../../../integration-tests/protractor.conf';
+import { click } from '../../../../console-shared/src/test-utils/utils';
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/utils/utils.ts]: Nit: we can import from `@console/shared` here.

Please go over any `../console-shared` (cross-package relative import) occurences in `kubevirt-plugin/integration-tests` and replace them with proper module import.

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts [hunk 2]
```diff
@@ -1,19 +1,44 @@
 import * as _ from 'lodash';
 import { $, browser, ExpectedConditions as until } from 'protractor';
-import { testName } from '../../../../integration-tests/protractor.conf';
-import { createNic, networkTypeDropdownId } from '../views/kubevirtDetailView.view';
+import { testName, appHost } from '../../../../integration-tests/protractor.conf';
+import {
+  createNic,
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts]: Nit: `createNIC`

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts [hunk 3]
```diff
@@ -56,58 +81,141 @@ describe('Add/remove disks and NICs on respective VM pages', () => {
...
+
+    await wizard.addNIC(multusNetworkInterface);
+    rowsCount = await wizardView.tableRowsCount();
+    await wizardView.activateTableRow(rowsCount);
+    await click($(wizardView.tableElementID('binding', rowsCount.toString())));
+    options = await getDropdownOptions(wizardView.tableElementID('binding', rowsCount.toString()));
+    expect(options.sort()).toEqual(nonPodNetworkBindingMethods); // expect masquerade is not available option
+  });
+
+  xit('BZ(1753232) Check nic binding method in example VM', async () => {
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts]: [Fix](https://github.com/openshift/console/pull/2919) for [BZ 1753232](https://bugzilla.redhat.com/1753232) is now merged, why `xit` here?

Also, nit: `NIC` :smiley:

**Comment by mareklibra** [comment]
The `ci/prow/analyze ` keeps failing on main vendor chunk size (approx. 3.8MiB locally).
The huge `react-icons` dependency is pulled in and causing this increase.

This PR is not adding it or touching (at least not directly). @spadgett , any idea, please?

**Pull request merged**